README.md

```
@@ -1,63 +1,100 @@
-```json
-{
-  "advanced": {
-    "http": true
-  },
-  "baseUrl": "http://localhost:3000",
-  "defaultPort": 3000
-}
+### Prerequisites
+
+- **Node.js**: v22.0.0
+- **PostgreSQL**: 16 (local install or Docker — see [Database setup](#database-setup))
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)